fleet/error.go

```diff
@@ -8,6 +10,18 @@ var (
 	maskAny = errgo.MaskFunc(errgo.Any)
 )
 
+func maskAnyf(err error, f string, v ...interface{}) error {
```

Comment:
I don't understand this function. Why can't the user just call `errgo.WithCausef(nil, err, f, v...)` ?